app/views/tracks/index.html.haml

```diff
@@ -6,38 +6,84 @@
+          %br
+          If something changes and you can't organize the track any more, you should withdraw it.
+
+        %p
+          %strong
+            Why do I need to add more information?
+        %p
+          The more information you add to your proposal, the more likely it is that the conference organizers accept your track request.
+
+        %p
+          %strong
+            Why do I need to register to the conference?
+        %p
+          Knowing the number of visitors for the conference helps the organizers plan better.
+
   - if @tracks.any?
     .row
       .col-md-12
-        %table.table.table-hover#tracks
-          %thead
-            %th Name
-            %th Short name
-            %th Description
-            %th Color
-            %th State
-            %th Actions
-          %tbody
-            - @tracks.each do |track|
-              %tr
-                %td
-                  = link_to(conference_program_track_path(@conference.short_title, track)) do
-                    = track.name
-                %td
-                  = track.short_name
-                %td
-                  %p
-                    = truncate(track.description)
-                %td
-                  %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
-                    = track.color
-                %td
-                  = track.state
-                %td
+        %table.table.table-striped#tracks
+          - @tracks.each do |track|
+            %tr
+              %td{style: "padding:15px 0px 0px 8px;"}
+                - if %w(new to_accept to_reject).include? track.state
+                  %span{ title: 'In review', class: 'fa fa-eye' }
+                - elsif track.state == 'accepted'
+                  %span{ title: 'Accepted', class: 'fa fa-check text-muted' }
+                - elsif track.state == 'confirmed'
+                  %spam{ title: 'Confirmed', class: 'fa fa-check text-success' }
+                - elsif %w(rejected withdrawn canceled).include? track.state
+                  %span{ title: track.state.humanize, class: 'fa fa-ban'}
+              %td
+                = link_to(conference_program_track_path(@conference.short_title, track)) do
+                  = track.name
+              %td
+                = markdown(truncate(track.description))
+              %td{style: "padding:15px 0px 0px 0px;"}
+                %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
+                  = track.color
+              %td
+                - if track.start_date
+                  From:
+                  = track.start_date.strftime('%A, %B %-d. %Y')
+              %td
+                - if track.end_date
+                  To:
+                  = track.end_date.strftime('%A, %B %-d. %Y')
+              %td.pull-right
```

Comment:
Something looks broken on the table (the horizontal line gets interrupted)

![screenshot from 2017-07-23 02-35-37](https://user-images.githubusercontent.com/4322737/28495232-bdf091ae-6f4f-11e7-9e31-5998be5cc4d5.png)